aleph0/.github/workflows/publish.yml

```diff
@@ -0,0 +1,19 @@
+name: "publish package to npm"
+
+on: push
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: node
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org
+      - name: publish
+        run: npm publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_AUTH_TOKEN}}
```

Comment:
doens't handle npm versions. think this fails unless package.json has updated version in it.

Reply:
I wanted to remove anyways :+1: